firefox_puppeteer/tests/test_about_window.py

```diff
@@ -4,9 +4,11 @@
 
 from marionette_driver import By
 
+from firefox_ui_harness.decorators import skip_under_xvfb
 from firefox_ui_harness import FirefoxTestCase
```

Comment:
For the future I would prefer to add sub-package inclusions below the upper package. But that's fine for now given that it is temporary anyway.